### PR TITLE
Link Analysis from home page

### PR DESCRIPTION
**Pull request by serras**
This tracks the progress of replacing Meta in the main page with Analysis:

- [X] Replace links and code blocks to Analysis
- [x] Add `arrow-analysis-lines` and `arrow-analysis-lines-white` versions of the SVGs in `arrow-site/docs/img/home` (temporarily using `color` for everything)

### CODE REVIEW

**Review by raulraja — APPROVED** [review]
thanks @serras !

File: arrow-site/docs/_code/fx-home-code.md [hunk 1]
```diff
@@ -26,10 +26,14 @@ fun employee(name: String, company: Company): Employee =
...
-    return candidates.parTraverse { hire(it) } //hires in parallel
+  // maps to 'Employee' in parallel using 'parZip'
+  val audrey = parZip({ "Audrey" }, { company("Arrow") }) { 
+    name, company -> Employee(name, company) 
+  } 
+  val pepe   = parZip({  "Pepe"  }, { company("Arrow") }) { 
+    name, company -> Employee(name, company) 
+  }
+  // hires in parallel using 'parTraverse'
+  return listOf(audrey, pepe).parTraverse { hire(it) }
```
Comment (nomisRev) [inline arrow-site/docs/_code/fx-home-code.md:37]: We need to check how this affects the website.

**Review by nomisRev — APPROVED** [review]
Thanks @serras !

File: arrow-site/docs/_code/fx-home-code.md (hunk 1 above)
Comment (serras) [reply arrow-site/docs/_code/fx-home-code.md:37]: I checked locally, and this snippet fits exactly the side of the code block, without any scroll bars (horizontal or vertical).

Comment (nomisRev) [reply arrow-site/docs/_code/fx-home-code.md:37]: Thanks so much for checking @serras!!

**Pull request merged**